root/static/scripts/common.js

```diff
@@ -1,3 +1,6 @@
+// IE 11 doesn't support Object.assign.
+require('core-js/modules/es6.object.assign');
```

Comment:
All it does is create a polyfill for `Object.assign` if it doesn't already exist. If it exists, it falls back to the native one. You can see the source code here: https://github.com/zloirock/core-js/blob/master/modules/es6.object.assign.js